index.php

```diff
@@ -1,2 +1,82 @@
-<?php header("Location: template.php"); ?>
-<!-- This will need to be addOrder.php once the file is created -->
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="styles/styles.css" rel="stylesheet" type="text/css" />
+    <script type="text/javascript" src="scripts/addorder.js"></script>
+    <link rel="icon" href="assets/icons/SiOverleaf.svg" type="image/icon" />
+    <title>GoToGro | Add Order</title>
+</head>
+
+<body>
+    <?php
+    include_once("navbar.inc");
+    ?>
+    <div class="addorder">
+        <h1 class="white-text">Add New Order</h1>
+        <form method="post" action="processORDER.php" novalidate="novalidate">
+            <div id="row_1">
+                <div class="inputField">
+                    <label for="product_ID1">Product ID</label>
+                    <select name="product_ID1" id="product_ID1">
+                        <option value="placeholder" hidden>Please Select</option>
+                        <option value="Milk">Milk</option>
+                        <option value="KitKat">KitKat</option>
+                        <option value="Milo">Milo</option>
+                        <option value="Cheese">Cheese</option>
+                        <option value="Ice Cream">Ice Cream</option>
+                    </select>
+                </div>
+                <div class="inputField">
+                    <label for="quantity1">Quantity </label>
+                    <input type="number" name="quantity1" id="quantity1" required="required" onblur="show_total()" />
+                </div>
+                <div class="inputField">
+                    <label for="price1">Price </label>
+                    <input type="number" name="price1" id="price1" required="required"
+                        onblur="show_total(); show_row2();" />
+                    <br>
+                </div>
```

Comment:
Any particular reason why the label and the type of field here are different to whats shown in Figma? I thought the intention was for the price to be hardcoded, although I do think being able to change the price during checkout is a good idea

Reply:
The "Total Price" for each thing, which is an AC for this ticket isn't here either, was that intended?